### PR TITLE
feat: initial implementation

### PR DESCRIPTION
**Pull request by KhudaDad414**
**Description:** 
* uses SSG for each page and only updates the data once a day(to not exceed the New Relic API limit)
* two metrics have been ignored in the dashboard because of the complication regarding the data collection.
Screenshot 2024-03-14 at 12 21 31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
-# asyncapi-metrics-dashboard 
+# AsyncAPI Metrics Dashboard
+
+This Next.js 14 application parses AsyncAPI metrics from New Relic and displays them for relevant groups, providing a dashboard for viewing these statistics. It employs a Static Site Generation (SSG) model, pre-rendering pages at build time, and updates the data only once daily to ensure optimal performance.
```
Comment (Amzani) [inline README.md]: You mean working groups? in this case we can link to this discussion https://github.com/orgs/asyncapi/discussions/1037 to give more context.

Comment (Amzani) [reply README.md]: `viewing these statistics` -> `viewing metrics`

**Review by Amzani — CHANGES_REQUESTED** [review]
We are not caching GraphQL queries? right?

**Comment by Amzani** [comment]
I've added the env variables in Netlify, however it doesn't fix the build yet:
https://app.netlify.com/sites/asyncapi-metrics-dashboard/deploys/65f3e1e9fa831612b0ab20f4

File: README.md (hunk 1 above)
Comment (KhudaDad414) [reply README.md]: Done.

**Comment by KhudaDad414** [comment]
> We are not caching GraphQL queries? right?

No. but we are rendering the pages as SSG so the effect should be the same. as I tested it.

**Comment by KhudaDad414** [comment]
@Amzani  I tried to publish it and it publishes on [Netlify](https://fabulous-shortbread-fae491.netlify.app/dashboard/DeveloperExperience/1M) and [Vercel](https://asyncapi-metrics-dashboard.vercel.app/dashboard/DeveloperExperience/1W) without any issues.

can you try a rebuild maybe?

**Review by Amzani — APPROVED** [review]

**Pull request merged**